exercises/protein-translation/canonical-data.json

```diff
@@ -118,7 +118,7 @@
           "expected": ["Methionine","Phenylalanine","Tryptophan"]
         },
         {
-          "description": "Translation stops if STOP codon present in short string",
+          "description": "Translation stops if STOP codon at end of sequence",
           "property": "translate",
           "strand": "AUGUUUUAA",
           "expected": ["Methionine","Phenylalanine", "STOP"]
```

Comment:
Should "STOP" be included in "expected"?  The stop example in the README didn't include stop in the resulting protein.

Reply:
This was added because of @cmccandless point (above) regarding consistency between tests. If "STOP" is expected in the tests for "STOP" codons, it might make sense to include them in the list of expected proteins?

Equally, it might not make too much sense - the Python track does not include them in protein lists, and only identifies them if they are input as a single codon.

Reply:
Perhaps then:
```text
"strand":  "UAA",
"expected": ""
```

Reply:
or update the README to list the STOP protein in the stop example....

Reply:
I think it would be better to expect an empty list in the case of `"expected": ["STOP"]`

Reply:
Ok, empty list coming up!